modules/kubernetes-addons/ingress-nginx/locals.tf

```diff
@@ -1,12 +1,13 @@
 locals {
   name = "ingress-nginx"
+  namespace = "nginx"
```

Comment:
Why are we changing this? Should be `ingress-nginx` as per https://kubernetes.github.io/ingress-nginx/deploy/#quick-start.

Reply:
Good question @askulkarni2. Per the [add-on documentation](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/nginx.md):

"Note that the ingress controller is deployed in the **nginx** namespace." 

So, it appears that there must be a reason for this to be done this way.

This is consistent with the error that was seen when deploy the add-on  before the change:

`Failed sync attempt to c0d70e7b110803dd967c91fec4cba9bd9622bcab: one or more objects failed to apply, reason: namespaces "nginx" not found (retried 1 times).`

 In fact this bug/change was introduced recently when PR #293 was merged 4 days ago. Before then the value for the namespace was indeed 'nginx' and not 'ingress-nginx'

Reply:
To me recollection, the error you are referring to was a mis-configuration of the addons repo. Is there an error you are seeing with `ingress-nginx`. Because again as per official documentation of the addon the namepsace is `ingress-nginx`. We will need to fix our documentation to reflect that.

Reply:
@camba1 I believe the place we need to update namespace is here https://github.com/aws-samples/eks-blueprints-add-ons/blob/main/chart/templates/ingress-nginx.yaml#L17

Reply:
Updated the repo above with the new namespace [PR](https://github.com/aws-samples/eks-blueprints-add-ons/pull/30) . Removed changes to the nginx locals and updated the doc to reflect the new namespace in this repo.
Please let me know if here is anything else that needs to be done